_pytest/_code/source.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import generators
+from __future__ import absolute_import, division, generators, print_function
```

Comment:
That was my initial reaction and did remove it, but to my surprise that import is [*actually used*](https://github.com/nicoddemus/pytest/blob/4d947077bbed4862d4e0a055ce967ba490be96af/_pytest/_code/source.py#L159):

```python
    def compile(self, filename=None, mode='exec',
                flag=generators.compiler_flag,
                dont_inherit=0, _genframe=None):
```

I did take a quick look at the docs for an alternative but couldn't find any, so decided to just let it be. Do you have a suggestion?